.releaserc.yml

```diff
@@ -21,11 +21,13 @@ plugins:
           release: patch
         - type: docs
           release: patch
+        - breaking: true
```

Comment:
Thats the [default](https://github.com/semantic-release/commit-analyzer/blob/7a2902e73b5aada0ab7a2781843a653b8033e0da/lib/default-release-rules.js#L7), isnt it?

Reply:
Yes, the default value is true but without explicitly set, the previous release didn't work. Maybe a preset that overrides this default value. Now it's working.

Reply:
I see thanks!